### ISSUE TITLE
Make it possible to scan long range advertisements

### ISSUE THREAD

**Issue by laurijamsa** [ruuvi/ruuvi.gateway_esp.c#58]
Make it possible to configure via web UI the nRF to scan BLE long range advertisements.

Just leaving this here for a reminder. Exact scanning intervals and other options to be defined later.

**Comment by ojousima** [comment]
Already done at least in theory, but needs to be tested. nRF52811 currently scans 50 % of time for BLE Long range, https://docs.ruuvi.com/ruuvi-gateway-firmware/untitled/gw-nrf52811-scanning . Currently missing is configuring the scan parameters over Web UI and writing any parameters to nRF52811.

**Comment by ojousima** [comment]
nRF52811 is not officially supported by S140 v6.1.1 which is used by Nordic SDK15.3, this task might require major updating on nRF52811. However, passing the configuration can be done over UART even if nRF52 disregards it.

**Comment by ojousima** [comment]
Removing milestone label 1.2 off this task as it depends on nRF52811.

**Comment by ojousima** [comment]
Done in ESP FW 1.2.0 + nRF52 FW 0.3.0
